homeassistant/components/unifi/device_tracker.py

```diff
@@ -167,17 +167,12 @@ async def async_will_remove_from_hass(self) -> None:
     def async_update_callback(self) -> None:
         """Update the clients state."""
 
-        @callback
-        def _make_disconnected(now):
-            """Mark client as disconnected."""
-            self._is_connected = False
-            self.cancel_scheduled_update = None
-            self.async_write_ha_state()
-
         if self.client.last_updated == SOURCE_EVENT:
+            is_wired = self.is_wired
+            event = self.client.event.event
```

Comment:
I think this is unnecessary

Reply:
I don't think it makes much difference.  I was trying to reduce the calls to `is_wired` but that really wasn't the issue as it was just showing up in the stack of the is_connected 518 * CLIENT_CONNECTED_ATTRIBUTES

Reply:
I reverted this.